tests/constants.py

```diff
@@ -15,6 +15,8 @@
_X\x07W\xa4!\x88i\x13\xa1>\xa7\xaba\x01\x11]g\xa1_\xc6h\x8cx~p\xa3U\x8e\x8f\x88&Y<\xff\x95Q\x1a=t\x1d\xea\xa2\xe4\xcau\x00_\xe68u8Q9\xb0\x8bu\xfa\x12PF\xae\xf6dd\\m\xa9\x18ia\x96(|\xbc+\xbd\xfbUa\xc5\x0b\x00\xaf\x02\x98MD\x93\xe4/\x18\xdaq\xc6\x9d\xd3\xe1\x0f\xe0~K]z@;`\xc1\xa8\xa8\xb0@\x84\xa8\x8f\r\xebo\xd0\xd0B*w\xfa\x85\x89k\xd3`\x9e\x13\x82\xae\x1e\xdbeP\xb3\xeb\x90da\xa9F\xd6Xo7uE\xa9\x8d\xc9DtsD\xb6b\x9aa\xbeY:/-\x1b\xab\nG\xe24@\xab\xbd\xbca\xfa\x8b=D\xa6\xd7\xae\xbatT\x00o\xcb\xae\x89h\xb2\xe5\x05\'*\xf3>\x16V\x9d\xb0x\xc1|*\x95\xea\xe1\xba\xee\x11j\x93\x84\n]T\xe3\x11\tD1J\x89\xd4q\x9c\x016\x8f\x14%$\x02H\xcb\xe4\xb6\xa5\xb1\xe2+\x1d[!G\r\x00\xb9\x80\xf2\x12\x18\x96\xe2B\x0c\xb5U\x0f\xc0\xec8\xfak\xd7 \xa2\x9b,\x1e\xe6\xa4\xb8\xf3\t\xfa\xa5\x14%\x1b-\x95J\xfd\x00\xec0\xc8\x16\xe4\x1b\xec8N\x7f\xe5y\x19\xc0JYF\xccg\x82$/\x89\xef\xe5\x9d\x10l5[\xa3\xa3\xa9\x06\xd9O4\x97\xb2\xc62\xf7\\MC\xf3\x10\x99\xaeR+M---\x07\xa8\xbd\x94\xa6Re@\x9a\x07l\x00\x00\x92\x0b\n\x9c\xf3s-{\xbb7\xecl\x05\xf7\xb9*\xae\xfeb\x0f\x91U}\x1a7\xd3\x0bIz\xc6[<\xe0\xb5\xcaKn\xb2\xbc\xe0\x1b\xca\x81\xcf\xd3Q(\x8a\xcc\xc2\x08\xa0\xfeE\x00\x8b\xe2@\x17\x00k\x0c\xf2\xcf\xaa]\xe2!\xbd\x9a\xbf\xd3\x18\xe5g\x8a\xccobt\x11\x8d\x97=\x19\x11M\x97/&\x11MW\xe6\xfa\x95E/w\x86\x9c\xeb8\x00\r")j\xecL\xe8\x177s>\xbf\x9da\xfcn\x1b\x15\xc1\x18\x1bCDSM4\x83Rw\xbf\x801v\n\x11\xfd\xd6\xc4\xedI\x87r[\xc8|\xcdD4E$O&B}\xa5!t\x99:\xb2\x9b8\xe7\xe73\xc6N\x03\x904E\n\x99G\xf5}\xbf\xdaq\x9cC\x89h\x9a\xda)o\xe8\xe2\xbf\xcf2\xdfb\xa1\xcf\xc9\xea%\xd7yg"\xba\xc32\xd7[\xe9t\xba\x97\xa6\xc3\xfe\\\x00\xaf\x068\x94\x88&Y:\xfcWv\x16E\xf4NggU6\x8eL\xb3\xce\x83&\x9a(\xe4w\xf5D4\x9d\x88\xa6\x00\xb8\xc7\xd6\t.J\xa9\x0b\x83,\x93\x88\xae\xb70\x00#t\x04u\xcc\xbd\xe5\xdb\xd9\xc7Y\xa3\xa3e\xd4N\xa3\x88s\xcd\x0e\xab\x00\x99\xf4*>\xf3X\n\xe0e\xa5\x0c\xb9\x9516\xd2\xd4\xe1o\xd2_{\x08\xf5\xd1\x96\x90s]\x07\x8c}VD\xe5\xcd\x13\xde\xa81\x82\xac#\xfe4\rb\x1e\xd0\x1a\xcf\xf3\x06+\xde\xcfD\xa8\xaf\xb7\xf4\x0f\x14#\xae7\xcbu\xdd\xa3L\r\r\x96\xe6\xd9\xbbL\x8e\xc0\xf4a\x9ca\xae"\x11M2\x9d\x8d\xad\xd9"d\x7f\xcbr\xb9\xdc\xbe!\x1d\xfe\xeb;\xcb[\xd6\x18\x16\xc8v$\xab\xf2}\x7f\xb8\xad\x1bF(oJX\xc6)\xc9\x7f\x10|\x9ea\x0bkj\xb6\xaa\xfb\xa3\x04\x9c\xf3\x0b,\xd0\xe5"\x8b\xae\x16D0\x88\x1fK\xf23"\xees3\xe7\xfc\xec\x10}\x1eo*m\xaa\xd8\xd9\xf4\xc9\x88\xf2.\xf7\xc40\xca\xb5r\x1b_{\xf5\x17y\x08\xca\xa4]\x1fzE\xcc\xf4/\x05\x90\xd1t\xc7<\xa2\x82c\x91I\xbf\xad\xb9\xa5\x1b\x88\xe8\xa7\x9a\xda\xec5\x00\xeau\xc6\x01\xe0u"\x9aj\xe1\n\xd7\xc5m\xa9\x93\x9ai\x17\xa9\x06"\xde\xe9\x0f:\xc0OD?\x01\x90\xd65p\x00XCDW\x9a**\xba^S\x00\xff4t\x1c-e\x8c\x8d\x8dy>S-\xe5\xcd\x8f\x00\xcc\x0f8\xdf\xce\xd0\xdf\xff\xd4H\xa7\xd3\xbd\x18c#9\xe7gz\x9e74\xec\x10<\xcf;\x96s~6cl\xb4\xfaG\x17\x0c\xf2G3\xc6\xbe\xc7\x18\x1b\xeby\xde\xb1r\x17xW\x8d\\.\xb7/cl4\xe7\xfc,\xdf\xf7\xab\xa3\xacY*\x95\xfa1\xc6\xc6p\xce\xcf\xf2<ohG\xa2\x91\xeb\xba\x838\xe7g0\xc6N-\x97\xcb\x03\xa3\x1a\xb6\xa5\xe2w8c\xecT\xf1>\xc3L_c\xee\xaa\xf1\x1f9\xec5\xd4\xfe\r\x9d\x05\x00\x00\x00\x00IEND\xaeB`\x82'
```

Comment:
I should find a better/smaller test image

Reply:
Like the smallest valid ELF, someone has done this: http://garethrees.org/2007/11/14/pngcrush/

And of course there's mathiasbynens/small which is the first google result for "smallest png"